containers/src/Data/Graph.hs

```diff
@@ -383,8 +383,8 @@ graphFromEdges' x = (a,b) where
     (a,b,_) = graphFromEdges x
 {-# INLINABLE graphFromEdges' #-}
 
--- | Build a graph from a list of nodes uniquely identified by keys,
--- with a list of keys of nodes this node should have edges to.
+-- | \(O((V+E) \log V)\). Build a graph from a list of nodes uniquely identified
```

Comment:
Isn't it $O(V + E \log V)$?

Reply:
The sort step is $O(V \log V)$, then the graph takes $O(V + E \log V)$ to build, so the total seems right to me.

Reply:
You're absolutely right. I got my edges and vertices crossed for a moment.